docs/team/hanjiyao.md

```diff
@@ -12,24 +12,30 @@ ContacX will allow for efficient management of a large list of contacts, allowin
 Given below are my contributions to the project.
 
 * **New Feature**: Bulk Operations
-  * What it does: `To be added soon`
-  * Justification: `To be added soon`
+  * What it does: Allow user to create multiple commands together by range batch or chain
+  * Justification: As part of target user profile, this is enhancement user experience to organise contacts
   * Highlights: `To be added soon`
   * Credits: `To be added soon`
 
-* **Code contributed**: `to be added soon`
+* **Code contributed**:
+  * 2000+ LoC contributed
```

Comment:
might wanna check the reposense for a accurate LoC estimate

Reply:
thx changed to more accurate